Packs/FireEyeCM/Integrations/FireEyeCM/FireEyeCM.py

```diff
@@ -344,15 +344,21 @@ def fetch_incidents(client: Client, last_run: dict, first_fetch: str, max_fetch:
     all_alerts = raw_response.get('alert')
 
     if not all_alerts:
-        demisto.info(f'{INTEGRATION_NAME} no alerts were fetched at: {str(last_run)}')
-        # as no alerts occurred till now, update last_run time accordingly
-        next_run['time'] = to_fe_datetime_converter('now')
+        demisto.info(f'{INTEGRATION_NAME} no alerts were fetched from FireEye server at: {str(next_run)}')
+        # as no alerts occurred in the window of 48 hours from the given start time, update last_run window to the next
+        # 48 hours. If it is later than now -10 minutes take the latter (to avoid missing events).
+        two_days_from_last_search = (dateparser.parse(next_run['time']) + timedelta(hours=48))
+        now_minus_ten_minutes = dateparser.parse('10 minutes').replace(tzinfo=two_days_from_last_search.tzinfo)
+        next_search = min(two_days_from_last_search, now_minus_ten_minutes)
+        next_run = {
+            'time': next_search.isoformat(),
+            'last_alert_ids': []
```

Comment:
why do we also set `last_alert_ids` to be empty list ?

Reply:
because we never clean that list, and in that case it ok to clean it.